src/main/java/com/google/api/codegen/discogapic/transformer/DiscoGapicNamer.java

```diff
@@ -167,15 +168,17 @@ public static Name getInterfaceName(String defaultInterfaceName) {
     return Name.anyCamel(Inflector.singularize(resource));
   }
 
-  /** Get the response type name from a method if the method has a non-null response type. */
+  /** Get the response type from a method if the method has a non-null response type. */
   @Nullable
-  public static Name getResponseName(Method method) {
+  public DiscoveryField getResponseField(Method method) {
```

Comment:
I'm a little confused - why would a response be a field? Doesn't a field have to be part of a message?

Reply:
The response types in GCE are message types.

Reply:
It's the terminology that's throwing me off - a method's attributes (e.g. request type, response type) aren't generally called "fields."

Reply:
Ok, renamed to getResponseType().